pages/upgrade-guide/3.5.md

```diff
@@ -38,7 +40,7 @@ would create the following structure:
 ```
 because `<div/>` was replaced with `<div></div>` & `<span/>` with `<span></span>`.
 
-jQuery 3.5.0 has changed `jQuery.htmlPrefilter` to be an identity function. That means that the above `jQuery` call would now create above HTML structure only in XML mode of HTML (called also as XHTML) but in regular HTML mode you would now get:
+jQuery 3.5.0 has changed `jQuery.htmlPrefilter` to be an identity function. In HTML mode auto-closing tags are being replaced with their opening tags, making `<div/><span/>` interpreted as `<div><span>`; the browser will automatically close these tags at the end of input. That means that the above `jQuery` call would now create above HTML structure only in XML mode of HTML (called also as XHTML) but in regular HTML mode you would now get:
```

Comment:
I'm wondering if people understand what "HTML mode" is nowadays. XHTML is pretty archaic!

I think the important point here is that the new interpretation of this string is being done by the browser's HTML5 parser and not jQuery.

Reply:
Yeah, I just didn't want to imply that `<div/><span/>` will **always** be converted to the nested structure by the browser as in XML mode it won't happen. Maybe I'll cut the XML part out.